packages/beacon-node/src/util/graffiti.ts

```diff
@@ -1,8 +1,37 @@
 import {GRAFFITI_SIZE} from "../constants/index.js";
+import {ExecutionEngineHttpOpts} from "../execution/engine/http.js";
+import {ClientCode, ClientVersion} from "../execution/index.js";
 
 /**
  * Parses a graffiti UTF8 string and returns a 32 bytes buffer right padded with zeros
  */
 export function toGraffitiBuffer(graffiti: string): Buffer {
   return Buffer.concat([Buffer.from(graffiti, "utf8"), Buffer.alloc(GRAFFITI_SIZE, 0)], GRAFFITI_SIZE);
 }
+
+export function getLodestarClientVersion(info?: ExecutionEngineHttpOpts): ClientVersion {
+  return {
+    code: ClientCode.LS,
+    name: "Lodestar",
+    version: info?.version ?? "",
+    commit: info?.commit?.slice(0, 4) ?? "",
+  };
+}
+
+export function getDefaultGraffiti(opts: {private?: boolean}, executionClientVersion?: ClientVersion): string {
+  if (opts.private) {
+    return "";
+  }
+
+  const consensusClientVersion = getLodestarClientVersion();
+
+  if (executionClientVersion != undefined) {
+    const {code: executionCode, commit: executionCommit} = executionClientVersion;
+
+    // Follow the 2-byte commit format in https://github.com/ethereum/execution-apis/pull/517#issuecomment-1918512560
+    return `${executionCode}${executionCommit.slice(0, 2)}${consensusClientVersion.code}${consensusClientVersion.commit}`;
```

Comment:
this should be 
```suggestion
    return `${executionCode}${executionCommit.slice(0, 4)}${consensusClientVersion.code}${consensusClientVersion.commit}`;
```

For example `LH1be5BU0f91`

Reply:
> this should be
> 
> For example `LH1be5BU0f91` (also CL info before EL info?)

There is no standardization on this. For example Lighthouse has EL first https://github.com/sigp/lighthouse/blob/9e12c21f268c80a3f002ae0ca27477f9f512eb6f/beacon_node/execution_layer/src/engine_api.rs#L817
whereas Teku has CL first https://github.com/Consensys/teku/blob/56a746b26366bff58c38da25213e7eb7800ea77f/beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/GraffitiBuilder.java#L145

Reply:
I don't see a lot of usage of this either, we can always adapt it later if format gets standardized, at some point someone trying to collect client diversity stats using graffiti will get pissed off by this but as I generally question the usefulness of using the graffiti for this, I am fine with using whatever format we have now as long as we don't leak this information by default like for example Teku does.